core/math/projection.cpp

```diff
@@ -876,6 +876,13 @@ bool Projection::is_orthogonal() const {
 	return columns[2][3] == 0.0;
 }
 
+bool Projection::is_frustum_symmetric() const {
+	return (columns[0][1] == 0.0f && columns[0][2] == 0.0f && columns[0][3] == 0.0f &&
```

Comment:
I have to find some time to look this up, but this check seems unlikely, this looks like it would be true only for identity matrices?

Reply:
I need to find some time to double check but laying it out like that, my gut feeling says yes. So I might have read the code wrong.

Reply:
Basically it's when left FOV = right FOV, top FOV = bottom FOV, so indeed `unslanted, unsheared, rectangular and unpanned.`